src/basic/basic.js

```diff
@@ -34,19 +34,19 @@ function squaredSum (numbers) {
  * @param {Array} numbers - An array of numbers
  * @returns {Number}
  */
-function mean (numbers) {
-  return sum(numbers) / numbers.length
+function mean(numbers) {
+  return sum(numbers) / numbers.length;
 }
 
-function sumOfSquaresOfDifference (numbers) {
-  const mean_val = mean(numbers)
+function sumOfSquaresOfDifference(numbers) {
+  const meanVal = mean(numbers);
 
   return numbers.reduce((acc, number) => {
-    const difference = number - mean_val
-    const squared = Math.pow(difference, 2)
+    const difference = number - meanVal;
+    const squared = square(difference);
```

Comment:
@johnpaulada , could I remove this variable "squared" or use it on the line below?